sdk/src/client/bindings/python/iota_client/client.py

```diff
@@ -79,17 +80,20 @@ def __init__(
                 nodes = [nodes]
         client_config['nodes'] = nodes
 
-        client_config = {k:v for k,v in client_config.items() if v != None}
+        client_config = {k: v for k, v in client_config.items() if v != None}
 
         def get_remaining_nano_seconds(duration: timedelta):
             return (int(duration/timedelta(microseconds=1))-int(duration.total_seconds())*1_000_000)*1_000
 
         if 'api_timeout' in client_config:
-            client_config['api_timeout'] = {'secs': int(client_config['api_timeout'].total_seconds()), 'nanos': get_remaining_nano_seconds(client_config['api_timeout'])}
+            client_config['api_timeout'] = {'secs': int(client_config['api_timeout'].total_seconds(
+            )), 'nanos': get_remaining_nano_seconds(client_config['api_timeout'])}
         if 'node_sync_interval' in client_config:
-            client_config['node_sync_interval'] = {'secs': int(client_config['node_sync_interval'].total_seconds()), 'nanos': get_remaining_nano_seconds(client_config['node_sync_interval'])}
+            client_config['node_sync_interval'] = {'secs': int(client_config['node_sync_interval'].total_seconds(
+            )), 'nanos': get_remaining_nano_seconds(client_config['node_sync_interval'])}
         if 'remote_pow_timeout' in client_config:
-            client_config['remote_pow_timeout'] = {'secs': int(client_config['remote_pow_timeout'].total_seconds()), 'nanos': get_remaining_nano_seconds(client_config['remote_pow_timeout'])}
+            client_config['remote_pow_timeout'] = {'secs': int(client_config['remote_pow_timeout'].total_seconds(
+            )), 'nanos': get_remaining_nano_seconds(client_config['remote_pow_timeout'])}
 
         client_config = humps.camelize(client_config)
```

Comment:
You mean the snake_case? :trollface:  Should I undo?

Reply:
But that's only for client?

Reply:
> But that's only for client?

For the ctr and the output builders

Reply:
So basically I tried to only keep the basic filtering and creation of an "alternative" dict functionality in the as_dict function and do everything else before sending the actual message to the message interface

Reply:
Not sure what you mean, how would it be camelized for `Address` for example?

Reply:
Doesn't need to do anything there since it's lowercase? https://github.com/iotaledger/iota-sdk/blob/9478df755a72c25721c7770a9669448272110f0a/sdk/src/client/bindings/python/iota_client/types/unlock_condition.py#L44

Reply:
I meant this https://github.com/iotaledger/iota-sdk/pull/256/files#diff-1e692a760a15c16b4152c90f4210b89a29f72abf87279e26862e247b31afafcfR30

Reply:
So we're asking the user to do this? Why?

Reply:
Are we still talking about the same thing? 😅